src/code/z_message_PAL.c

```diff
@@ -3170,8 +3170,7 @@ void Message_Update(PlayState* play) {
                     R_TEXTBOX_TEXHEIGHT = 512;
                 } else {
                     Message_GrowTextbox(msgCtx);
-                    // TODO: this may be NA_SE_PL_WALK_GROUND - SFX_FLAG, or not, investigate sfxId=0
-                    Audio_PlaySfxIfNotInCutscene(0);
+                    Audio_PlaySfxIfNotInCutscene(NA_SE_NONE);
```

Comment:
Do you think this was (un)intended?

Reply:
There was a textbox growing sound effect in beta (can hear it in some spaceworld era vids)

They likely just put a raw 0 in there to "comment it out" without removing the whole call

Reply:
It's hard to say, I think Fig hypothesized that these were likely actual sounds in earlier releases (sw97) and that the sounds were too obnoxious or not right, so they put a `NONE` here to silence this sfx (maybe there was a debug conditional)? But I kinda lean into that kind of concept, so I suspect this was intentional and they were messing around with different sfx at this spot, but the final decision was to not include one

Reply:
lmao. Beat me to the punch